src/openrct2-ui/windows/EditorScenarioOptions.cpp

```diff
@@ -1121,6 +1134,29 @@ static void WindowEditorScenarioOptionsParkMousedown(rct_window* w, rct_widgetin
             }
             w->Invalidate();
             break;
+        case WIDX_FORBID_HIGH_CONSTRUCTION_SPINNER_INCREASE:
+            if (gConstructionHeightRestriction < 118)
```

Comment:
Is there a constant that better represents what this limit is?

Reply:
Ah, yeah, I had intended to go look for one, but kind of forgot. It's supposed to be the max build height stated in height units. Do you know of one?